### ISSUE TITLE
I'm having trouble compiling SBVA

### ISSUE THREAD

**Issue by cacajr** [hgarrereyn/SBVA#6]
First of all, congratulations on your work!

I'm trying to use your preprocessor to perform some tests on files in DIMAC format, but I'm having trouble. I use Linux Ubuntu 22.04.4 LTS and what I've done so far is:

1 - Cloned your repository
2 - Downloaded and unzipped Eigen inside the repository
3 - Downloaded clang version 14.0.0-1ubuntu1.1
4 - Ran the "make" command inside the repository

After all this, I received the following error:

``` CMD
clang++ -I /eigen-3.4.0/ -std=c++11 -O3 -o sbva sbva.cc
sbva.cc:2:10: fatal error: 'cstdlib' file not found
#include <cstdlib>
^~~~~~~~~
1 error generated.
make: *** [Makefile:7: sbva] Error 1
```

Do you know what I'm doing wrong?

**Comment by cacajr** [comment]
Just to complement what I've already done so far:

5 - I modified the Makefile file pointing to the full path of the clang compiler and the Eigen directory.

**Comment by ndrewh** [comment]
It seems like you might not have a working C++ compiler. I just added a Dockerfile, which includes all of the steps necessary to build our tool on Ubuntu 22.04:

https://github.com/hgarrereyn/SBVA/blob/acc0a5170beb63e9cd6f8449c735396c802c7d8b/Dockerfile#L1-L8

**Comment by cacajr** [comment]
Thank you very much for the response! Using containers in these cases avoids the famous 'it works on my machine,' haha.

However, I was intrigued by the fact that your Dockerfile describes exactly the steps I took. I looked into what might be happening and found the solution here: [link](https://askubuntu.com/questions/1449769/clang-cannot-find-iostream). I installed g++-12 and it worked.

Thanks for the support!
